Data-Collection-Inventory/2021Q1_csvs/grant.csv

```diff
@@ -1,32 +1,20 @@
 grant_reference_id,grant_title,grant_identifier,funding_agency_reference_id,awardee_reference_id,report_symbol
 Allen Institute funder,Allen Institute,Allen Institute,Allen Institute for Brain Science,Allen Institute for Brain Science,
-Autism_EC grant,No title,supplement  to  #4987,Autism Speaks,University of California at San Diego,
-BrainSpan_EL_MH grant,TRANSCRIPTIONAL ATLAS OF HUMAN BRAIN DEVELOPMENT,1RC2MH089921-01,National Institute of Mental Health,Allen Institute for Brain Science,
-BrainSpan_JK grant,TRANSCRIPTIONAL ATLAS OF HUMAN BRAIN DEVELOPMENT,1RC2MH090047-01,National Institute of Mental Health,University of Southern California,
-BrainSpan_NS grant,TRANSCRIPTIONAL ATLAS OF HUMAN BRAIN DEVELOPMENT,1RC2MH089929-01,National Institute of Mental Health,Yale University,
-R24 Ghosh grant,DANDI: Distributed Archives for Neurophysiology Data Integration,1R24MH117295-01A1,National Institute of Mental Health,Massachusetts Institute of Technology,9795271
-R24 Ropelewski grant,A Confocal Fluorescence Microscopy Brain Data Archive,1R24MH114793-01,National Institute of Mental Health,Pittsburgh Supercomputing Center,9567623
```

Comment:
keep

Reply:
Added BICCN-related grants and subprograms that don't have associated projects yet.